lib/hanami/cli/commands/app/generate/repo.rb

```diff
@@ -30,13 +30,9 @@ def generator_class
 
             # @since 2.2.0
             # @api private
-            def call(name:, slice: nil, **opts)
-              normalized_name = if name.end_with?("_repo")
-                                  name
-                                else
-                                  "#{inflector.singularize(name)}_repo"
-                                end
-              super(name: normalized_name, slice: slice, **opts)
+            def call(name:, **opts)
+              name = "#{inflector.singularize(name)}_repo" unless name.end_with?("_repo")
```

Comment:
Just checking: by reassigning `name` here, that new value will still be passed up to the superclass method via the no-args version of `super` below?

Reply:
Yup! Which I wasn't expecting. I wrote it explicitly as `super(name: name, **opts)`, but my Rubocop autocorrect got rid of it since it's equivalent to `super`.